.github/workflows/nix.yml

```diff
@@ -57,7 +60,7 @@ jobs:
           echo ${PATH} | tr ':' '\n'
           conan --version
           cmake --version
-          env
+          env | sort
       - name: configure Conan
         env:
           CONAN_URL: http://18.143.149.228:8081/artifactory/api/conan/conan-non-prod
```

Comment:
if the dns mapping changes in the future, won't we be in trouble because we have hard-coded the IP address?

Reply:
There is no domain name for this address. No mapping to change.

The risk is that the public IP address for this machine changes. We have already discussed, internally, adding a domain name for this machine. It will be an improvement, but it is not necessary.

Lastly, the IP address is already hard-coded in the workflows. It is not part of this PR.

Reply:
I see, ok